apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -9,30 +9,45 @@ const styles = {
     justifyContent: 'space-between',
     flexDirection: 'row',
     alignItems: 'center',
+    height: '100%',
+  },
+  overviewMain: {
+    padding: 10,
   },
   icon: {
     color: color.level_perfect,
   },
-  text: {
+  value: {
     color: color.charcoal,
     fontFamily: '"Gotham 5r", sans-serif',
     marginRight: 10,
   },
 };
 
+function  calculateOpacity(answered) {
+ return (answered + 10)/100;
+}
+
 class MultipleChoiceAnswerCell extends Component {
   static propTypes = {
     percentValue: PropTypes.number.isRequired,
     isCorrectAnswer: PropTypes.bool,
     displayAnswer: PropTypes.string,
+    isSurvey: PropTypes.bool,
   };
 
   render() {
-    const {percentValue, isCorrectAnswer, displayAnswer} = this.props;
+    const {percentValue, isCorrectAnswer, displayAnswer, isSurvey} = this.props;
+
+    const opacity = calculateOpacity(percentValue);
+
+    const backgroundCSS = (isCorrectAnswer || isSurvey) ? {backgroundColor: `rgba(159, 212, 159, ${opacity})`} :
```

Comment:
you could pull the logic on lines 42-45 into a function like this:
```
getBackgroundColor = (percentValue) => {
  const {isCorrectAnswer, isSurvey} = this.props;
  const opacity = calculateOpacity(percentValue);
  return (isCorrectAnswer || isSurvey) ? `rgba(159, 212, 159, ${opacity})` : `rgba(255, 99, 71, ${opacity})`;
};
```
then plug that into: `const backgroundCSS = {backgroundColor: getBackgroundColor(percentValue};` to clarify the logic here a little. not a blocker, but something to consider

Reply:
Great idea, will follow up.